packages/lit-renderer/src/lit-renderer.d.ts

```diff
@@ -0,0 +1,14 @@
+import { RenderOptions } from 'lit';
```

Comment:
Let's add license header in files under `src` folder.

Reply:
Done.